extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -39,7 +39,7 @@
 			a.async = 1;
 			a.src = g;
 			m.parentNode.insertBefore(a, m);
-		})(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+		})(window, document, 'script', 'https//www.google-analytics.com/analytics.js', 'ga');
```

Comment:
I guess that won't work because there might miss a colon after `https` and before `//`.